guides/common/modules/proc_creating-custom-rhel-images.adoc

```diff
@@ -6,7 +6,7 @@
 In this example, we use a {RHEL} 7 Workstation.
```

Comment:
We should really update this to use RHEL 9.

Reply:
Please, really file a Jira issue.